### ISSUE TITLE
Missing decimal point in topic payload

### ISSUE THREAD

**Issue by Joseraj** [arendst/Tasmota#9499]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

Since yesterday mqtt payload doesn't has decimal points, at least for temperature and humidity:
```
20:09:19 ZIG: {"ZbEZSPReceived":"45000004010000010100010000DAA8C6B0F2FFFF2C185C0A01FF42250121090B0421A8430521B700062400000000006429F3066521711B662B4A8C01000A210000"}
20:09:19 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":0,"srcaddr":"0xF2B0","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":76,"securityuse":0,"seqnumber":218,"fc":"0x18","manuf":"0x0000","transact":92,"cmdid":"0x0A","payload":"01FF42250121090B0421A8430521B700062400000000006429F3066521711B662B4A8C01000A210000"}}
20:09:19 ZIG: ZbZCLRawReceived: {"0xF2B0":{"0000/FF01":"0121090B0421A8430521B700062400000000006429F3066521711B662B4A8C01000A210000","Endpoint":1,"LinkQuality":76}}
20:09:19 MQT: tele/Outdoor_TempHumPres/SENSOR = {"ZbReceived":{"Outdoor_TempHumPres":{"Device":"0xF2B0","Name":"Outdoor_TempHumPres","0000/FF01":"0121090B0421A8430521B700062400000000006429F3066521711B662B4A8C01000A210000","BatteryVoltage":28.25,"BatteryPercentage":62,"Temperature":1779,"Humidity":7025,"Pressure":1015,"SeaPressure":1015,"Endpoint":1,"LinkQuality":76}}}
```

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [forum](https://groups.google.com/d/forum/sonoffusers)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): zbbridge
- [x] Tasmota binary firmware version number used: 

Program Version | 9.0.0.1(zbbridge)
Build Date & Time: 2020-10-08T13:00:01
 Core/SDK Version: 2_7_4_3/2.2.2-dev(38a443e)

  - [x]  Pre-compiled
  - [ ] Self-compiled
    - [ ] IDE / Compiler used: _____
- [ ] Flashing tools used: _____
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```
  Configuration output here:
20:06:22 CMD: Backlog Template; Module; GPIO 255
20:06:22 SRC: WebConsole from 192.168.2.248
20:06:22 CMD: Group 0, Index 1, Command "BACKLOG", Data "Template; Module; GPIO 255"
20:06:22 SRC: Backlog
20:06:22 CMD: Group 0, Index 1, Command "TEMPLATE", Data ""
20:06:22 MQT: stat/ZbBridge/RESULT = {"NAME":"Sonoff ZbBridge","GPIO":[320,3552,0,3584,5312,0,0,0,0,576,0,0,32,0],"FLAG":0,"BASE":75}
20:06:22 SRC: Backlog
20:06:22 CMD: Group 0, Index 1, Command "MODULE", Data ""
20:06:22 MQT: stat/ZbBridge/RESULT = {"Module":{"75":"Sonoff ZbBridge"}}
20:06:22 SRC: Backlog
20:06:22 CMD: Group 0, Index 1, Command "GPIO", Data "255"
20:06:22 MQT: stat/ZbBridge/RESULT = {"GPIO0":{"320":"Led_i1"},"GPIO1":{"3552":"Zigbee Tx"},"GPIO2":{"0":"None"},"GPIO3":{"3584":"Zigbee Rx"},"GPIO4":{"5312":"Zigbee Rst"},"GPIO5":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"640":"I2C SDA"},"GPIO13":{"576":"LedLink_i"},"GPIO14":{"608":"I2C SCL"},"GPIO15":{"0":"None"},"GPIO16":{"32":"Button1"},"GPIO17":{"0":"None"}}

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```
  Rules output here:


```
- [x] Provide the output of this command: `Status 0`:
```
  STATUS 0 output here:
20:07:44 CMD: Status 0
20:07:44 SRC: WebConsole from 192.168.2.248
20:07:44 CMD: Group 0, Index 1, Command "STATUS", Data "0"
20:07:44 MQT: stat/ZbBridge/STATUS = {"Status":{"Module":75,"DeviceName":"ZbBridge","FriendlyName":["ZbBridge"],"Topic":"ZbBridge","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0}}
20:07:44 MQT: stat/ZbBridge/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/tasmota-zbbridge.bin","RestartReason":"Software/System restart","Uptime":"0T01:52:55","StartupUTC":"2020-10-08T16:14:49","Sleep":50,"CfgHolder":4617,"BootCount":139,"BCResetTime":"2020-08-07T22:54:08","SaveCount":404,"SaveAddress":"FA000"}}
20:07:44 MQT: stat/ZbBridge/STATUS2 = {"StatusFWR":{"Version":"9.0.0.1(zbbridge)","BuildDateTime":"2020-10-08T13:00:01","Boot":31,"Core":"2_7_4_3","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"385/699"}}
20:07:44 MQT: stat/ZbBridge/STATUS3 = {"StatusLOG":{"SerialLog":0,"WebLog":3,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["Pepe",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C8000100060000005A00000000000000","00008000","40006082","00000000"]}}
20:07:44 MQT: stat/ZbBridge/STATUS4 = {"StatusMEM":{"ProgramSize":532,"Free":468,"Heap":23,"ProgramFlashSize":1024,"FlashSize":2048,"FlashChipId":"1540A1","FlashFrequency":40,"FlashMode":3,"Features":["00000809","0F100786","04500001","00000002","00000000","00000000","00020000","00000000"],"Drivers":"1,2,4,7,9,10,20,21,23,41","Sensors":""}}
20:07:44 MQT: stat/ZbBridge/STATUS5 = {"StatusNET":{"Hostname":"ZbBridge-2578","IPAddress":"192.168.2.83","Gateway":"192.168.2.1","Subnetmask":"255.255.255.0","DNSServer":"1.1.1.1","Mac":"C8:2B:96:06:4A:12","Webserver":2,"WifiConfig":4,"WifiPower":17.0}}
20:07:44 MQT: stat/ZbBridge/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.2.63","MqttPort":1883,"MqttClientMask":"ZbBridge","MqttClient":"ZbBridge","MqttUser":"openhabian","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30}}
20:07:44 MQT: stat/ZbBridge/STATUS7 = {"StatusTIM":{"UTC":"2020-10-08T18:07:44","Local":"2020-10-08T20:07:44","StartDST":"2020-03-29T02:00:00","EndDST":"2020-10-25T03:00:00","Timezone":99,"Sunrise":"07:21","Sunset":"18:39"}}
20:07:44 MQT: stat/ZbBridge/STATUS10 = {"StatusSNS":{"Time":"2020-10-08T20:07:44"}}
20:07:44 MQT: stat/ZbBridge/STATUS11 = {"StatusSTS":{"Time":"2020-10-08T20:07:44","Uptime":"0T01:52:55","UptimeSec":6775,"Vcc":3.517,"Heap":23,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"Pepe","BSSId":"AC:9E:17:7B:72:E0","Channel":4,"RSSI":76,"Signal":-62,"LinkCount":1,"Downtime":"0T00:00:03"}}}

```
- [ ] Provide the output of the Console log output when you experience your issue; if applicable:
  _(Please use_ `weblog 4` _for more debug information)_
```
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

Just upgrade via Ota

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

Report payload as previously

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
The UI reports numbers right
![Immagine 2020-10-08 201310](https://user-images.githubusercontent.com/16203682/95497566-d05c6480-09a2-11eb-9d6c-3c30b428c7ba.png)

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by ascillato2** [comment]
Hi,

Have you tried command `tempres`?

**Comment by Joseraj** [comment]
Hi,
Ho I haven't, but if I understood the docs that command sets the resolution as digits after the decimal point, but not the position of the decimalpoint.
I didn't change anything in last week, just done OTA update.
Probably configuration is right, as it is showing right on the UI.

Anyway thank for evidence that command, I think resolution higher than 0.1 is useless for temperature or humidity.

**Comment by ascillato** [comment]
Ok, but please, try command `tempres 1` in the Tasmota console.

**Comment by Joseraj** [comment]
Just tryed, 
```06:17:03 ZIG: {"ZbEZSPReceived":"45000004010000010100010000017CBBB0F2FFFF2C187C0A01FF42250121090B0421A8430521B800062400000000006429B8056521B920662BFC8A01000A210000"}
06:17:03 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":0,"srcaddr":"0xF2B0","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":47,"securityuse":0,"seqnumber":1,"fc":"0x18","manuf":"0x0000","transact":124,"cmdid":"0x0A","payload":"01FF42250121090B0421A8430521B800062400000000006429B8056521B920662BFC8A01000A210000"}}
06:17:03 ZIG: ZbZCLRawReceived: {"0xF2B0":{"0000/FF01":"0121090B0421A8430521B800062400000000006429B8056521B920662BFC8A01000A210000","Endpoint":1,"LinkQuality":47}}
06:17:03 MQT: tele/Outdoor_TempHumPres/SENSOR = {"ZbReceived":{"Outdoor_TempHumPres":{"Device":"0xF2B0","Name":"Outdoor_TempHumPres","0000/FF01":"0121090B0421A8430521B800062400000000006429B8056521B920662BFC8A01000A210000","BatteryVoltage":28.25,"BatteryPercentage":62,"Temperature":1464,"Humidity":8377,"Pressure":1011,"SeaPressure":1011,"Endpoint":1,"LinkQuality":47}}}
```
Nothing changed, and dont see any change in resolution.

It seems that the sensor sends an ingegrer "1464" that should be divided by 100.
It is divided by 100 at some point because the ui shows "14.6", but not before publishing on mqtt.
![Immagine 2020-10-10 062346](https://user-images.githubusercontent.com/16203682/95645633-6d152400-0ac1-11eb-9493-ddb3ef47af00.png)

Now I've seen that "BatteryVoltage" has the decimal point, but in wrong place.
```BatteryVoltage":28.25``` should be ```BatteryVoltage":2.825```

**Comment by s-hadinger** [comment]
Sorry I missed this issue. Can you please paste the result of the command `ZbStatus2 0xF2B0`

I want to check ModelId and Manufacturer.

**Comment by Joseraj** [comment]
Xiaomi WSDCGQ11LM
```
12:19:03` CMD: ZbStatus2 0xF2B0
12:19:03 SRC: WebConsole from 192.168.2.22
12:19:03 CMD: Group 0, Index 2, Command "ZBSTATUS", Data "0xF2B0"
12:19:03 MQT: stat/ZbBridge/RESULT = `{"ZbStatus2":[{"Device":"0xF2B0","Name":"Outdoor_TempHumPres","IEEEAddr":"0x00158D0002786760","ModelId":"lumi.weather","Manufacturer":"LUMI","Endpoints":[1]}]}
```

**Comment by s-hadinger** [comment]
Bug introduced in #9491, fixed with #9515
